TU20Bot/Configuration/Controllers/AuthenticationController.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using EmbedIO;
+using EmbedIO.Routing;
+using EmbedIO.WebApi;
+using Jose;
+using MongoDB.Driver;
+using TU20Bot.Models;
+
+namespace TU20Bot.Configuration.Controllers {
+    public class AuthenticationController : ServerController {
+
+        [Route(HttpVerbs.Post, "/login")]
+        public string Login([QueryField] string username, [QueryField] string password) {
+
+            var user = server.client.database.GetCollection<AccountModel>(AccountModel.collectionName).Find(_user => _user.username.Equals(username.ToLower()));
```

Comment:
```suggestion
            var user = server.client.database
                .GetCollection<AccountModel>(AccountModel.collectionName)
                .Find(x => x.username == username.ToLower());
```
For long lines like this it's typically good to split them.